web/src/components/Content.tsx

```diff
@@ -54,12 +55,17 @@ export const Content = (props: ContentProps): ReactElement => {
       return <HorizontalLine />;
     },
     blockquote: GreenBox,
-    greenBox: (props: any): ReactElement => {
-      return <GreenBox>{props.children}</GreenBox>;
-    },
     note: (props: any): ReactElement => {
       return <Alert variant="note">{props.children}</Alert>;
     },
+    callout: (props: any): ReactElement => {
+      return (
+        <Callout header={props.header} icon={props.icon} calloutType={props.calloutType}>
```

Comment:
[sand] Let's turn `icon` into a boolean. You could do something like: 
```suggestion
        <Callout header={props.header} icon={props.icon === "true"} calloutType={props.calloutType}>
```